plugins/module_utils/_api_data.py

```diff
@@ -2808,6 +2808,18 @@ def join_path(path):
             'test-id': KeyInfo(default=0),
         },
     ),
+    ('user',): APIData(
+        primary_keys=('name', ),
+        fully_understood=True,
+        fields={
+            'address': KeyInfo(),
+            'comment': KeyInfo(can_disable=True, remove_value=''),
+            'disabled': KeyInfo(default=False),
+            'group': KeyInfo(),
+            'name': KeyInfo(),
+            'password': KeyInfo(),
```

Comment:
Hmm, I guess password is write-only and cannot be read? We probably have to adjust `KeyInfo()` to allow write-only fields, otherwise this can quickly lead to unexpected behavior.

I'm currently modifying `_api_data.py`, so it's probably better to wait with this until I'm done, otherwise this will be a conflict fest ;-)

Reply:
Yes, `password` is write-only. In what sense do we need to change `KeyInfo`?

Reply:
I'll try out something tomorrow.